2DR.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link href="2dr.css" rel="stylesheet" >
+</head>
+<body>
```

Comment:
Try to be more sematic. Avoid using `div` all the time. Chips may be replaced with `buttons` and such.